src/modules/src/system.c

```diff
@@ -201,23 +201,38 @@ void systemTask(void *arg)
 #endif
 
   //Test the modules
+  DEBUG_PRINT("About to run tests in system.c (one fail causes the following to fail).\n");
   pass &= systemTest();
+  DEBUG_PRINT("system %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= configblockTest();
+  DEBUG_PRINT("configblock %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= storageTest();
+  DEBUG_PRINT("storage %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= commTest();
+  DEBUG_PRINT("comm %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= commanderTest();
+  DEBUG_PRINT("commander %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= stabilizerTest();
+  DEBUG_PRINT("stabilizer %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= estimatorKalmanTaskTest();
+  DEBUG_PRINT("estimatorKalmanTask %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= deckTest();
+  DEBUG_PRINT("deck %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= soundTest();
+  DEBUG_PRINT("sound %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= memTest();
+  DEBUG_PRINT("mem %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= watchdogNormalStartTest();
+  DEBUG_PRINT("watchdogNormalStart %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= cfAssertNormalStartTest();
+  DEBUG_PRINT("cfAssertNormalStart %s\n", pass ? "[OK]" : "[FAIL]");
   pass &= peerLocalizationTest();
+  DEBUG_PRINT("peerLocalization %s\n", pass ? "[OK]" : "[FAIL]");
```

Comment:
These are a lot of prints on the console. I have 2 worries about it:
 - It is a lot more data printed at startup which is going to accumulate in the console buffer. It should still be ok but the risk of adding too much is to overflow the bugger
 - More important, the fact that if one test fails, all the subsequent one will be shown as failed which can be very confusing to the use.

One proposal would be to instead only show the failed tests:
```
if (systemTest() == false) {
  pass = false;
  DEBUG_PRINT("system [FAIL]");
}
```

It is more code but it would alleviate both my concerns. Another idea would be to show the printout only in debug mode (this is what we do for the deck driver, we are much more verbose in debug mode).